packages/manager/apps/pci-workflow/src/pages/list/actions.component.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  OsdsButton,
+  OsdsIcon,
+  OsdsMenu,
+  OsdsMenuItem,
+  OsdsText,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_VARIANT,
+  ODS_ICON_NAME,
+  ODS_ICON_SIZE,
+  ODS_TEXT_LEVEL,
+} from '@ovhcloud/ods-components';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import { useHref } from 'react-router-dom';
+
+type ActionsProps = {
+  projectId: string;
+  backupId: string;
+};
+export default function Actions({ backupId }: Readonly<ActionsProps>) {
+  const { t } = useTranslation();
+  const hrefExecutions = useHref(`./${backupId}`);
+  const hrefDelete = useHref(`./delete?workflowId=${backupId}`);
+
+  return (
+    <OsdsMenu>
```

Comment:
You can use the actionMenu of manager-components to make it easy 

here an example 
```
const items = [
    {
      id: 0,
      href: hrefEdit,
      label: t('pci_projects_project_storages_blocks_edit_label'),
      dataTestid: 'actionComponent-edit-button',
    },
    {
      id: 1,
      href: volume.attachedTo?.length ? hrefDetach : hrefAttach,
      label: t(
        `pci_projects_project_storages_blocks_instance_${
          volume.attachedTo?.length ? 'detach' : 'attach'
        }_label`,
      ),
      dataTestid: 'actionComponent-attach-detach-button',
    },
    {
      id: 2,
      href: hrefCreateBackup,
      label: t('pci_projects_project_storages_blocks_create_backup_label'),
      dataTestid: 'actionComponent-create-backup-button',
    },
    {
      id: 3,
      href: hrefRemove,
      label: t('pci_projects_project_storages_blocks_delete_label'),
      dataTestid: 'actionComponent-remove-button',
    },
  ];

  return <ActionMenu items={items} isCompact />;
```